src/libs/ontuml2gufo/relation_gufo_inverse.ts

```diff
@@ -0,0 +1,79 @@
+import { N3Writer } from 'n3';
+import {
+  NormalRelationStereotypeMapping,
+  InverseRelationStereotypeMapping,
+} from './constants';
+
+const N3 = require('n3');
+const { DataFactory } = N3;
+const { namedNode, quad } = DataFactory;
+
+export async function transformGUFOInverses(
+  writer: N3Writer,
+): Promise<boolean> {
+  const objectProperPartInverserList = [
+    'hasCollectionMember',
+    'hasComponent',
+    'hasSubCollection',
+    'hasSubQuantity',
+  ];
+  const quads = [
+    quad(
+      namedNode('gufo:hasObjectProperPart'),
+      namedNode('rdfs:subPropertyOf'),
+      namedNode('gufo:hasProperPart'),
```

Comment:
We cannot use the `gufo` prefix because these properties are not defined there. We should use base prefix that has been provided.